desktop/ui/src/main/java/org/datacleaner/util/RemoteServersConfigUtils.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.util;
+
+import org.apache.metamodel.util.Resource;
+import org.datacleaner.configuration.DataCleanerConfiguration;
+import org.datacleaner.configuration.DataCleanerConfigurationImpl;
+import org.datacleaner.configuration.DataCleanerConfigurationUpdater;
+import org.datacleaner.repository.RepositoryFile;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+/**
+ * Utils for work with information about remote servers in configuration file
+ */
+public class RemoteServersConfigUtils {
```

Comment:
I think actually we should avoid using this class (and DataCleanerConfigurationUpdater) because they stray away from the unified changes to conf.xml that we recently tried to implement in the class ```DomConfigurationWriter```. It would be best to only update conf.xml in a single way, which that class does well right now for datastores, dictionaries, synonym catalogs and string patterns. I would prefer if we simply add some remote component server awareness to that class.